docs/page-objects/index.rst

```diff
@@ -0,0 +1,121 @@
+.. _page-objects:
+
+============
+Page objects
+============
+
+A page object is a code wrapper for a webpage that implements the logic to
```

Comment:
It may be also a wrapper for a part of webpage.

Reply:
https://github.com/scrapinghub/web-poet/pull/152/commits/73f59eb278d85b895a8279e7cd26e97f17d2b002